src/main/java/net/dancier/dancer/chat/ChatService.java

```diff
@@ -0,0 +1,15 @@
+package net.dancier.dancer.chat;
+
+import net.dancier.dancer.chat.dto.ChatDto;
+import org.springframework.stereotype.Service;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+
+@Service
+public class ChatService {
+    public List<ChatDto> getChatsByUserId(UUID userId) {
```

Comment:
At this part, I would like to have only DancerIds. The relates to what we just talked about some days ago:
I would think that we better have the dependencies to this kind of authentication system, not cluttered of all business code. But how can we achieve this. I also did that in the rest of the dancer. Maybe we find no solution before the MVP, or maybe my concerns are not valid at all ;-)

Reply:
Yes, I thought entering the method with the userId and then looking up the dancer id. We could also have some kind of interceptor which is triggered after authentication and then looks up the dancer id. We could use the AuthenticationSuccessHandler and then looking up the dancer id and add it to the request (if it is possible at this step).